spec/requests/api/v1/payments_spec.rb

```diff
@@ -0,0 +1,316 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+
+RSpec.describe 'payments API', type: :request do
+  let!(:user_params) do
+    {
+      "email": 'fake_email@fake_email.com',
+      "full_name": 'Oliver Twist',
+      "greeting_name": 'Oliver',
+      "language": 'English',
+      "password": 'password1234!',
+      "password_confirmation": 'password1234!',
+      "phone_number": '912-444-5555',
+      "phone_type": 'home',
+      "organization": 'Society for the Promotion of Elfish Welfare',
+      "service_agreement_accepted": 'true',
+      "timezone": 'Central Time (US & Canada)'
+    }
+  end
+  let(:user_id) { User.create(user_params).id }
+  let!(:agency_params) do
+    {
+      "name": 'Agency 1',
+      "state": 'IL'
+    }
+  end
+  let(:agency_id) { Agency.create!(agency_params).id }
+  let(:business_params) do
+    {
+        "name": 'Happy Hearts Child Care',
+        "category": 'licensed_center_single',
+        "user_id": user_id
+    }
+  end
+  let(:business_id) { Business.create!(business_params).id }
+  let!(:site_params) do
+    {
+      "name": 'Evesburg Educational Center',
+      "address": '1200 W Marberry Dr',
+      "city": 'Gatlinburg',
+      "state": 'TN',
+      "zip": '12345',
+      "county": 'Harrison',
+      "qris_rating": '4',
+      "business_id": business_id
+    }
+  end
+  let(:site_id) { Site.create!(site_params).id }
+  let!(:payment_params) do
+    {
+      "agency_id": agency_id,
+      "amount_cents": '123400',
```

Comment:
Couple questions:

Do the `amount_cents` and `discrepancy_cents` fields come in as parameters or should it come in as `amount` when hitting the api?

If it should be `amount_cents` and `discrepancy_cents`, do we also need to pass `amount_currency` and `discrepancy_currency`?

Reply:
Hm.  Good questions.  I'm not sure; will investigate.

Reply:
I checked -- only need to use the attribute name.  Only the `money-rails` gem needs to use the `_cents` and `_currency` fields.  (Unless you're doing something specific.  We're not.)

I'll update accordingly.

Reply:
OK ...  you _do_ need to specify the `_cents`   (Makes more sense that you do.  Then it's very clear that the `integer` that is being sent to/from literally is an amount is _cents._)